python/chronos/src/bigdl/chronos/forecaster/autoformer_forecaster.py

```diff
@@ -185,10 +316,13 @@ def fit(self, data, epochs=1, batch_size=32):
                               batch_size=batch_size,
                               shuffle=True)
 
+        from bigdl.chronos.pytorch import TSTrainer as Trainer
         # Trainer init and fitting
-        self.trainer = Trainer(logger=False, max_epochs=epochs,
-                               checkpoint_callback=self.checkpoint_callback, num_processes=1,
-                               use_ipex=self.use_ipex, distributed_backend="spawn")
+        if not self.use_hpo:
+            self.trainer = Trainer(logger=False, max_epochs=epochs,
+                                   checkpoint_callback=self.checkpoint_callback, num_processes=1,
+                                   use_ipex=self.use_ipex, distributed_backend="spawn")
```

Comment:
what will happen if users call `fit` but use same configspace during the initialization? If it's not allowed, we may give clear error message here.

Reply:
done. Add a check and a ut for this case.